sdks/python/apache_beam/version.py

```diff
@@ -39,20 +42,9 @@ def get_version_from_pom():
     pattern = re.compile(str(regex))
     search = pattern.search(pom)
     version = search.group(1)
-    version = version.replace("-SNAPSHOT", ".dev")
-    # TODO: PEP 440 and incubating suffix
+    version = version.replace("-incubating-SNAPSHOT", ".dev+incubating")
```

Comment:
@sb2nov has the updated PR (https://github.com/apache/beam/pull/2187) .I was just referencing this as a starting point. Otherwise this is old :)